.yarn/plugins/@yarnpkg/plugin-git-hooks.cjs

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable */
```

Comment:
Just checking that these are supposed to be pushed?

Reply:
Yes.